invenio_records_permissions/config.py

```diff
@@ -13,3 +14,20 @@
     "invenio_records_permissions.policies.RecordPermissionPolicy"
 )
 """PermissionPolicy for records."""
+
+RECORDS_PERMISSIONS_READ_ONLY = False
```

Comment:
minor: the description indicates that the type of the variable could be `bool`, `str` or `callable`. is it really necessary for the feature to provide all three possibilities?

Reply:
the short answer is "no", it's not necessary... but it _does_ offer a bit of flexibility

i would argue that the `str` and `callable` versions are almost identical (the import `str` provides easy configuration via environment variables, while the `callable` alternative might be more desirable in `invenio.cfg` or from other python code), so it basically comes down to the two options `str/callable` vs. `bool`.

the latter option (`bool`) was the original implementation in our own module and had the benefit of being dead simple – either the system is in RO-mode or not (also indicated a bit by the configuration name).
however, changing the system's mode would require a configuration reload (e.g. via redeploy) which might be a bit of a hassle if it needs to be done regularly (e.g. if there's a weekly maintenance window).
as such, i added the other possibility to give more flexibility to those admins who "know what they're doing".